yarn.lock

```diff
@@ -512,7 +512,78 @@
     protobufjs "^6.11.2"
     uint8arrays "^3.0.0"
 
-"@chainsafe/persistent-merkle-tree@^0.4.2":
+"@chainsafe/lodestar-api@^0.37.0":
+  version "0.37.0"
+  resolved "https://registry.yarnpkg.com/@chainsafe/lodestar-api/-/lodestar-api-0.37.0.tgz#d1e6d0d313af9c8f782db8fd437040f862daf545"
+  integrity sha512-9WWvxpAG2bcnfK9s5krn8d4249AtYRfAcC10RquoQYyayNTP67Arx023Zid0t3bpFH1I8MOTgGDVeGr1HuhPxQ==
+  dependencies:
+    "@chainsafe/abort-controller" "^3.0.1"
+    "@chainsafe/lodestar-config" "^0.37.0"
+    "@chainsafe/lodestar-params" "^0.37.0"
+    "@chainsafe/lodestar-types" "^0.37.0"
+    "@chainsafe/lodestar-utils" "^0.37.0"
+    "@chainsafe/persistent-merkle-tree" "^0.4.1"
+    "@chainsafe/ssz" "^0.9.1"
+    cross-fetch "^3.1.4"
+    eventsource "^1.1.0"
+    qs "^6.10.1"
+
+"@chainsafe/lodestar-beacon-state-transition@^0.37.0":
```

Comment:
Please what the ** is up with this yarn.lock diff showing up everywhere??? :rofl:

Reply:
It's perplexing to me too 😬. It shows up when I run `yarn install`. Probably something wrong with my local set-up. I'll take a look into what could be going wrong.

Reply:
@dadepo should be fixed by #4189